db/migrate/20170519153350_add_showback_models.rb

```diff
@@ -0,0 +1,73 @@
+class AddShowbackModels < ActiveRecord::Migration[5.0]
+  def up
+    create_table :showback_usage_types, :id => :bigserial, :force => :cascade do |t|
+      t.string     :category, :allow_nil => false
+      t.string     :description
+      t.string     :measure, :allow_nil => false
```

Comment:
@sergio-ocon Please remove all of the db/schema.rb cruft: `:id => bigserial, :force => cascade, :allow_nil, :default`, etc.